packages/connect-agreement/src/entities/Signature.ts

```diff
@@ -0,0 +1,19 @@
+import { SignatureData, IAgreementConnector } from '../types'
+
+export default class Signature {
+  #connector: IAgreementConnector
+
+  readonly id!: string
+  readonly signerId!: string
+  readonly versionId!: string
+  readonly createdAt!: string
```

Comment:
I think we could remove the `!`s here (they disable the non-null check, which shouldn’t be needed since we assign them in the constructor).